pkg/apis/config/types.go

```diff
@@ -142,4 +158,11 @@ type NodeResourceTopologyMatchArgs struct {
 
 	// Namespaces to be considered by NodeResourceTopologyMatch plugin
 	Namespaces []string
+
+	// ScoringStrategy a scoring model that determine how the plugin will score the nodes.
+	ScoringStrategy *ScoringStrategy
+
+	// Resources a list of pairs <resource, weight> to be considered while scoring
+	// allowed weights start from 1.
+	Resources       []schedulerconfig.ResourceSpec
```

Comment:
The formatting doesn't look right. Did you have a "run gofmt on every save" setup on your editor?